package.json

```diff
@@ -75,6 +75,23 @@
         "react-dom": ">=18",
         "stylelint": ">=15"
       }
+    },
+    "overrides": {
+      "platform@0.4.0 ≤ Version ≤ 1.3.5": "",
+      "merge@< 2.1.1": "2.1.1",
+      "glob-parent@< 5.1.2": "5.1.2",
+      "platform@1.3.3": "",
+      "jsbn@≤ 1.1.0": "",
+      "@cypress/request@< 3.0.0": "3.0.0",
+      "ejs@3.1.9": "",
+      "semver@< 5.7.2||6.0.0 ≤ Version < 6.3.1||7.0.0 ≤ Version < 7.5.2": "5.7.2||6.3.1||7.5.2",
+      "merge@<2.1.1": ">=2.1.1",
+      "glob-parent@<5.1.2": ">=5.1.2",
+      "socket.io-parser@>=4.0.4 <4.2.3": ">=4.2.3",
+      "@cypress/request@<=2.88.12": ">=3.0.0",
+      "semver@<5.7.2": ">=5.7.2",
+      "semver@>=7.0.0 <7.5.2": ">=7.5.2",
+      "postcss@<8.4.31": ">=8.4.31"
```

Comment:
Result of `pnpm audit --fix` to fix dependabot alerts: https://github.com/swisspost/design-system/security/dependabot

Reply:
Is this something we should do regularly, also to maintain this newly added section? 

If yes, can you extract this change and write a simple GitHub action that runs once a week or month and creates a PR after running this command?

Reply:
I'm not sure what's best. Let's forget it at the moment and try to fix those vulnerabilities by updating packages.